server/src/main/java/org/apache/iotdb/db/rescon/SystemInfo.java

```diff
@@ -54,6 +56,11 @@ public class SystemInfo {
   private long flushingMemTablesCost = 0L;
   private AtomicLong compactionMemoryCost = new AtomicLong(0L);
 
+  private AtomicInteger compactionFileNumCost = new AtomicInteger(0);
+
+  private int maxFileNumForCompaction =
+      config.getMaxCrossCompactionCandidateFileNum() * config.getCompactionThreadCount();
```

Comment:
Can we use a totalNum in simple rather than calculated by CompactionThreadCount ?

Reply:
Actually we already have a hard limitation to make total file count in one cross task is no more than getMaxCrossCompactionCandidateFileNum(), is this var `maxFileNumForCompaction` necessary ?